_summaries/sutskever2014sequence_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: stskever2014sequence
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: saikm200022
+score: 9
 ---
 
-TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+# **Summary - Sequence to Sequence Learning with Neural Networks, Sutskever, Vinyals, Le; 2014**
+
+## What is the core idea?
+
+Deep Neural Networks are being used today for all kinds of difficult tasks such as image detection and speech recognition. However, DNNs are not suitable for sequence to sequence tasks because the dimensionality of input and output sequences can be arbitrary. This paper suggests a Deep Learning approach that is able to apply DNNs for a sequence to sequence task by bypassing its limitation of requiring fixed dimensionality input and output.
+
+![Alt Text](sutskever2014sequence_1b.PNG)
+
+
+## How is it realized (technically)?
+
+The model was used on the WMT'14 English to French Machine Translation task. The paper trained the model on a subset of 12 million sentences with vocabulary size of 348 million and 304 million words for French and English respectively. 
+
+The model in the paper uses two LSTMs: one for the input and output sequences respectively. These LSTMs were relatively deep with four layers and a final softmax layer over the LSTMs output. The paper's training algorithm maximized the following to obtain the most likely translation: 
+
+![Alt Text](sutskever2014sequence_1a.PNG)
+
+The most likely transalations during training were found using a left-to-right beam search algorithm. The paper also took the 1000 most likely translations from the baseline (SMT) and rescored them using the log probabilities of each translation hypothesis with their model. The model was trained using Stochastic Gradient Descent without momentum with a batch size of 128 random sentences. 
+
+## How well does the paper perform?
+
+They assessed the success of their approach using a BLUE score - typically computed by comparing a translated sentence with some reference translation checking for some overlap. The higher the score, the closer the model's output translations were to the reference translations. With 5 reversed LSTMs and a beam size of 12 they were able to achieve a BLEU score of 34.81, which was larger than what was achieved by the SMT baseline. The paper claims this was the first time that such results ever occurred for a neural translation model. The specific results of their experiment are shown below:
```

Comment:
The LSTM reversion should be lifted to the section above to at least tell the audience what it is.

Reply:
I added the reverse property of the input LSTM to the model architecture above. Hopefully, this removes ambiguity when its referred in this section.